webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTableFormats.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+export const hostsCountFormat = (value, { rowData: { name } }) => (
+  <a href={`hosts?search=model+%3D+"${encodeURI(name)}"`}>{value}</a>
+);
+
+export const actionsFormat = (value, { rowData: { name, id } }) =>
+  (value.includes('delete') ? (
+    <span className="btn btn-sm btn-default">
+      <a
+        data-confirm={__(`Delete ${name}`)}
+        data-id={`aid_models_${id}-${encodeURI(name)}`}
+        rel="nofollow"
+        data-method="delete"
+        href={`/models/${id}-${encodeURI(name)}`}>
+        Delete
+      </a>
+    </span>
+  ) : null);
+
+export const nameFormat = (value, { rowData: { actions, id, name } }) =>
+  (actions.includes('edit') ? (
+    <a
+      href={`models/${id}-${encodeURI(name)}/edit`}
+      data-id={`aid_models_${id}-${encodeURI(name)}_edit`}>
+      {value}
+    </a>
+  ) : (
+    <a href="#" className="disabled" disabled="disabled" onClick={() => false}>
+      {value}
+    </a>
+  ));
+
+const orderChars = {
+  ASC: '▲',
+  DESC: '▼',
+  undefined: '',
+};
+
+const orderAPI = {
+  ASC: 'DESC',
+  DESC: 'ASC',
```

Comment:
?? our API is opposite ?

Reply:
The name is misleading. Basically, if now the order is "ASC" then in the next call we should set the order to "DESC" that's what I meant.

Anyway, I understand the confusion and will fix it.